py/selenium/webdriver/firefox/firefox_profile.py

```diff
@@ -53,13 +53,13 @@ def __init__(self, profile_directory=None):
            This defaults to None and will create a new
            directory when object is created.
         """
+        self._strict_timestamps = False
         self._desired_preferences = {}
         if profile_directory:
-            newprof = os.path.join(tempfile.mkdtemp(), "webdriver-py-profilecopy")
+            self._profile_dir = os.path.join(tempfile.mkdtemp(), "webdriver-py-profilecopy")
             shutil.copytree(
-                profile_directory, newprof, ignore=shutil.ignore_patterns("parent.lock", "lock", ".parentlock")
+                profile_directory, self._profile_dir, ignore=shutil.ignore_patterns("parent.lock", "lock", ".parentlock")
```

Comment:
I think we want to keep old logic here. The intent is to copy the directory then assign _profile_dir to the new location.

Reply:
I reverted the changes. I don't understand the need to use an intermediate variable, but it seems minor.

Reply:
I'm not certain either, stuff like this I assume it fixed someone's bug at some point and consider it a "Chesterton's Fence."  I'd look at commit history if it was a big deal or if if it seemed to cause performance issues. In general people shouldn't be relying on Firefox Profiles any longer (the whole class/approach was deprecated at one point, but I removed that because it is still technically supported). We do need to do a lot more deprecation work in these (there's an open issue somewhere).

Reply:
I suggest leaving this piece of code as it is. If the FirefoxProfile functionality is already deprecated, it may eventually be removed on its own in the future. For now, it indeed seems minor. I just noticed the long chain of assignments.